content/themis/languages/nodejs/installation.md

```diff
@@ -226,3 +226,64 @@ Error: Key Pair generation failed: invalid parameter
   code: 12
 }
 ```
+
+### Verified example how to fix it.
```

Comment:
what we are suggesting here? We show example of dockerfile that works. But our main goal is to describe how to build themis from the sources.  Why we shod dockerfile instead of commands for ubuntu/debian how to do it? Or if we rely on compilation using docker containers then let's describe how it use then. How to copy from the docker container and place into the target system